vivisect/base.py

```diff
@@ -695,3 +706,31 @@ def _cb_branchtable(self, tablebase, tableva, destva):
     
         return True
 
+    def addEntryPoint(self, va, arch=envi.ARCH_DEFAULT):
```

Comment:
slightly different codeflow for Viv, allowing for arch-modifying function addresses.